data'/Ineva-A.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Regina Ganser",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "WebDev, DevRel. Motivated to Programm, self learner",
+  "avatar": "https://github.com/Ineva-A.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/ineva_yt",
+      "icon": "FaTwitter"
+    }
```

Add `,` after each object
```suggestion
    },
```